penthouse.gemspec

```diff
@@ -32,8 +32,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'ar-octopus', '~> 0.8.6'
   spec.add_development_dependency 'activesupport', '~> 4.2.6'
   spec.add_development_dependency 'activerecord', '~> 4.2.6'
-  spec.add_development_dependency 'pg'
-  
+  spec.add_development_dependency 'pg', '~> 0.21.0'
```

Comment:
Set to mirror the platform, v1.0.0+ is incompatible with this version of ActiveRecord.